rasa/nlu/registry.py

```diff
@@ -30,7 +30,7 @@
 from rasa.nlu.featurizers.sparse_featurizer.count_vectors_featurizer import (
     CountVectorsFeaturizer,
 )
-from rasa.nlu.featurizers.dense_featurizer.lm_featurizer import LanguageModelFeaturizer
+from rasa.nlu.featurizers.dense_featurizer._lm_featurizer import LanguageModelFeaturizer
```

Comment:
this change shouldn't be needed. Let's do it as it's done [here](https://github.com/RasaHQ/rasa/blob/4eda9168a6231ae104531c231205c78b0bd1fb97/rasa/nlu/featurizers/sparse_featurizer/lexical_syntactic_featurizer.py#L42)